frontend/test/features/home/home_page_test.dart

```diff
@@ -67,7 +69,12 @@ void main() {
 
     testWidgets('should show empty state when no transactions', (tester) async {
       await tester.pumpWidget(
-        WidgetHelpers.testableWidget(child: HomePage()),
+        WidgetHelpers.testableWidget(
+          child: const HomePage(),
+          overrides: [
+            transactionsProvider.overrideWith((ref) => []),
+          ],
```

Comment:
as mentioned in [this comment](https://github.com/TBD54566975/didpay/pull/42/files#r1475423436), overriding list of transactions to return `[]` to show empty state